react/tic-tac-toe/src/app/screens/LogIn/components/Field/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import cx from 'classnames';
+
+const getValidityClassName = meta => {
+  if (meta.asyncValidating){
+    return 'async-validating';
```

Comment:
I'd create a couple of constants in a constants file for all these strings

Reply:
:white_check_mark: